### DISCUSSION TITLE
天翼云盘添加报错

### DISCUSSION THREAD

**Discussion by krwu** [AlistGo/alist#1158] [Q&A] [answered]
不管是 189cloud 还是 189cloudpc，添加时都报告错误，用189cloud的话是设备id不存在，189cloudpc的话是“设备ID不存在，需要二次设备校验”。

有人遇到同样的问题吗？有什么解决的办法吗？

**Comment by tlze** [comment, marked as answer]
文档-常见问题：（建议）
“被风控了，改密码再试”

**Reply by smshijiea** [reply]
谢谢，修改密码后登录成功。

**Comment by zhlx2835** [comment]
改密码也不行，是怎么回事

**Reply by Smile8848** [reply]
> 改密码也不行，是怎么回事

天翼云签-20250617近期登录失败的解决方法 - 吾爱破解 - 52pojie.cn
https://www.52pojie.cn/forum.php?mod=viewthread&tid=1999099&highlight=%CC%EC%D2%ED%D4%C6  关设备锁

**Reply by qcnhy** [reply]
关不了设备锁 手机客户端还能登录 可以扫码 也可以扫码登陆网页端 获取cookie  但是就是没法在alist使用 需要二次认证，这个情况有办法么 没办法修改密码或关闭设备锁
